test/driver.js

```diff
@@ -269,6 +269,8 @@ var Driver = (function DriverClosure() { // eslint-disable-line no-unused-vars
     this.inFlightRequests = 0;
     this.testFilter = parameters.testFilter ?
       JSON.parse(parameters.testFilter) : [];
+    this.coverageparam = parameters.coverageparam ?
```

Comment:
You have renamed `coverageparam` to `coverage`. `coverage` is not an array, and not JSON-stringified, so you should just use `this.coverage = parameters.coverage;` here.

Reply:
oh ok thanx